data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	You ask a nearby Heliarch agent where Pairri will be competing, and they check a computer before telling you that of the twenty floors, she's on the sixteenth. They point you towards the nearest elevator. After a ride as crowded as the train is, you rush to find a vacant seat. Though a quick glance shows there are likely enough seats on this level alone for over a hundred thousand spectators, you don't want to risk having to fight for a space given how quickly the seating area is filling up with fans all eagerly waiting for the event to begin. As you settle in, you spot many groups of Arachi and several lone Saryds, but compared to the thousands upon thousands of Kimek crowded around them, from a distance they look nearly as out of place as you feel yourself. `
+			`	After a few minutes, bridges extend from just under the seating area of the next level above you to the circular platform in the middle of the room. Hundreds of Kimek contestants then walk out upside-down, clinging to the underside of the bridges and platform. Heliarch agents divide them into two teams, numbering several hundred contestants each. The Heliarchs leave, and, after a long whistle is blown, the two teams of hanging Kimek charge at one another with surprising speed and agility, despite their swarming numbers and inverted orientation.`
+			`	The pile formed by the colliding teams looks like somewhat like a collective sumo match as the teams attempt to dislodge their opponents from the platform. Within a few seconds, dozens of Kimek have been knocked or outright thrown to the net below, where they are helped out to the sides by workers. Looking through the net, you can see that this is being repeated on each floor. Another whistle blows after enough have fallen out, and the remaining Kimek stop fighting. Heliarch agents return to the ring and organize those left once more into two teams. They depart, and the whistle blows again. This time the fighters have more space to maneuver, so almost a minute passes before any fall.`
+			`	This process repeats several more times until finally, those that remain - less than a tenth of those that entered - are gathered in the center. As they leave the ring, without an announcement of any rankings or score, you become unsure if the game has ended.`
+			`	You feel something tap the top of your head and turn to find that it is an Arach sitting behind you. "Confused, the human visitor is?" their little box translates.`
+			choice
+				`	"Yes. I have no idea what is going on."`
+				`	"I think I somewhat understand, but an explanation would be helpful."`
+			`	They break down the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins, when one hundred remain."`
```

Comment:
```suggestion
			`	They break down the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins when one hundred remain."`
```
This is a little strange. Regardless of which way you flip that last sentence, it's still structured normally - and if normal grammar rules apply (which I think they should because it's a normal sentence), that comma shouldn't be there.